src/atlas.nim

```diff
@@ -86,12 +91,12 @@ Options:
   --showGraph           show the dependency graph
   --keepWorkspace       do not update/overwrite `atlas.workspace`
   --list                list all available and installed versions
-  --version             show the version
+  --version / -v        show the version
   --ignoreUrls          don't error on mismatching urls
   --verbosity=normal|trace|debug
                         set verbosity level to normal, trace, debug
-  --global              use global workspace in ~/.atlas
-  --help                show this help
+  --global / -g         use global workspace in ~/.atlas
+  --help / -h           show this help
```

Comment:
Consider using the standard `-s, --short` or `-s=VAL, --short=VAL` notation.

I guess it's a bit less suitable due to [parseopt.nim idiosyncrasies](https://github.com/nim-lang/Nim/issues/12425), such as requiring a `=`/`:` even for short options taking values (thus requiring repeating the argument name in the help string) but it's still a neater presentation, especially if aligned properly.

Reply:
That's nicer, I'll switch to it.